CONTRIBUTING.rst

```diff
@@ -251,3 +252,71 @@ To view them locally
     (klio-docs) $ make clean && make livehtml
 
 Then navigate to ``http://localhost:8888`` in your browser.
+
+
+Microsite
+---------
+
+The Klio `microsite/landing page <https://spotify.github.io/klio>`_ is a simple static site that uses `Bootstrap 4.5 <https://getbootstrap.com/docs/4.5/getting-started/introduction/>`_.
+For now, it is not managed by any tooling (npm, gulp, etc); HTML, CSS, etc is maintained by hand.
+
+
+Local Development
+^^^^^^^^^^^^^^^^^
+
+1. In ``microsite/index.html``, add ``<script src="js/reload.min.js"></script>`` to the bottom after the ``</body>`` closing tag.
+
+.. caution::
+
+    Be sure to remove this addition when committing changes.
```

Comment:
Someday, I'd like to have a proper tooling ([for example](https://getbootstrap.com/docs/4.0/getting-started/build-tools/)) to maintain this site rather than having to manage this ish ourselves.